src/pycatcher/diagnostics.py

```diff
@@ -333,8 +337,8 @@ def build_moving_average_outliers_plot(df: pd.DataFrame) -> plt:
     lower_bound = df1['moving_average'] - 2 * df_pandas.iloc[:, -1].rolling(window=optimal_window_size).std()
 
     # Identify outliers
-    outliers = df1[(df1['above_threshold'] == True) | (df1['below_threshold'] == True)].dropna()
-    return_outliers = outliers.iloc[:, 1]
+    outliers = df1[(df1['above_threshold']) | (df1['below_threshold'])].dropna()
+    # return_outliers = outliers.iloc[:, 1]
```

Comment:
Not using this return_outliers. Need to reassess if we need it at all

Reply:
Updated code to remove it